packages/kaizen-tg/_templates/drupal-9-theme/new/packages/components/_variables.css

```diff
@@ -2,10 +2,56 @@
 to: <%= h.src() %>/<%= h.changeCase.lower(name) %>/packages/components/_variables.css
 ---
 :root {
-  --root-font-size: 2vw;
-  --base-font-size: 16px;
+  /* font-sizes */
+  --font-size--xxxxxl: 48px;
```

Comment:
All these custom font-sizes are project specific. Shouldn't be here

Reply:
maybe keep them. but they shouldn't be used by components. 
and also reduce their amount.

Reply:
it is kind of every project starting settings. so few vars of that type can be here. just as boilerplate